Solving Issue #202 (#1509)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/collaboration-exo/public-folders/set-up-public-folders.md

> [!NOTE]
> For more information about the storage quotas and limits for public folders, see [Exchange Online Limits](https://go.microsoft.com/fwlink/p/?LinkID=391188).

> [!IMPORTANT]

> > The procedure in this article guided you through the process of creating Public Folders the 1st time.

Reviewer: ```suggestion
> The procedure in this article guides you through the process of creating public folders for the first time.
```

The public folder will be mail-enabled and the name of the public folder will become the alias of the public folder. If you have multiple recipients with that name, the public folder's alias will be appended with a number. For example, if you have a distribution group named SalesTeam and you create a public folder named SalesTeam and then mail-enable it, the alias of that public folder will be SalesTeam1.

For information about how to use Exchange Online PowerShell to mail-enable a public folder, see [Enable-MailPublicFolder](https://technet.microsoft.com/library/6fc7ba9a-62a8-4f41-811f-608363aa1397.aspx).

> [!NOTE]

> > In case that you have a hybrid configuration, the Public folders created on Exchange Online are visible for only Mailboxes on Cloud. Oppositely, Public folders created on-Premises are visible for Mailboxes on the On-premises.

Reviewer: ```suggestion
> If you have a hybrid configuration, the public folders created on Exchange Online are only visible to cloud-based mailboxes. Conversely, public folders created on-premises are only visible to on-premises mailboxes.
```

> [!TIP]

> > In case that finished a migration from Exchange Server 2010 to Exchange Online with public folders, follow the following article:

Reviewer: ```suggestion
> To complete a migration from Exchange Server 2010 to Exchange Online with public folders, see [Configure legacy on-premises public folders for a hybrid deployment](https://docs.microsoft.com/exchange/collaboration-exo/public-folders/set-up-legacy-hybrid-public-folders).
```

> > [Configure legacy on-premises public folders for a hybrid deployment](https://docs.microsoft.com/exchange/collaboration-exo/public-folders/set-up-legacy-hybrid-public-folders)

Reviewer: ```suggestion
```
